R/features.R

```diff
@@ -87,11 +115,104 @@ feature_exact <- function(m, weight_zero_m = 10^6) {
   dt[, features := unlist(combinations, recursive = FALSE)]
   dt[, n_features := length(features[[1]]), id_combination]
   dt[, N := .N, n_features]
-  dt[, shapley_weight := shapley_weights(m = m, N = N, n_features, weight_zero_m)]
+  dt[, shapley_weight := shapley_weights(m = m, N = N, n_components = n_features, weight_zero_m)]
+
+  return(dt)
+}
+
+#' @keywords internal
+group_fun <- function(x, group_num) {
+  if (length(x) != 0) {
+    unlist(group_num[x])
+  } else {
+    integer(0)
+  }
+}
+
+#' Analogue to feature_exact, but for groups instead.
+#'
+#' @inheritParams shapley_weights
+#' @param group_num Vector of characters. Contains the feature labels used by the model
+#'
+#' @return data.table with all feature group combinations, shapley weights etc.
```

Comment:
Not to mention it was completely wrong :-) Thanks!

Reply:
Oh nice. But I actually meant for the @return data.table. Since you say group combinations, shapley weights etc. But remove etc and just write what is returned.

Reply:
Ah, I see now that is where your comment is. Sure, it could be. It is an internal function so it is really just a question of whether it is helpful for us when working on the code. It is an analogue to feature_exact which has no documentation at all, and it is all right there in the code. I might be biased though, so what do you think?

Reply:
Right. I didn't realize it was internal. Then I think it's fine the way it is.